axonhub-grpc-proto/src/main/proto/platform_api.proto

```diff
@@ -103,10 +104,11 @@ message EventProcessorInfo {
 
     string processor_name = 1;
     string mode = 2;
-    int32 activeThreads = 3;
-    bool running = 4;
-    bool error = 5;
-    repeated EventTrackerInfo event_trackers_info = 6;
+    int32 active_threads = 3;
```

Comment:
Backwards compatibility issue. When running this client version against previous server version, the server will not be able to parse the messages (as GRPC uses the sequence numbers). 
Better to change the sequence nr for available threads to 7.